recipes-core/glibc-locale-tests/glibc-locale-tests.bb

```diff
@@ -0,0 +1,70 @@
+SUMMARY = "NI-LinuxRT-specific glibc locale tests"
+HOMEPAGE = "https://github.com/ni/meta-nilrt"
+BUGTRACKER = "https://github.com/ni/meta-nilrt/issues"
+SECTION = "tests"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+DEPENDS = "glibc"
+
+
+SRC_URI = "\
+	file://run-ptest \
+	file://test_locale_aliases.c \
+	file://test_locale_aliases.sh \
+	file://CP932.txt \
+	file://CP936.txt \
+	file://L1.txt \
+"
+
+S = "${WORKDIR}"
+
+
+inherit ptest
+
+CC += " ${LDFLAGS}"
+debugsrcdir = "/usr/src/debug/${BPN}"
+
+do_compile() {
+	cd ${S}
```

Comment:
I don't think this line is required. Ref [usleep_1.0.bb](https://github.com/ni/meta-openembedded/blob/nilrt/master/kirkstone/meta-oe/recipes-core/usleep/usleep_1.0.bb#L15)

Reply:
I prefer the `cd ${S}` and usage of `${S}` over directly using the `WORKDIR` as it allows us to move the source files into a different location and only have to change the value of `S`. I'd prefer to leave it as is unless you object strongly.